iceoryx_examples/iceperf/iceperf_laurel.cpp

```diff
@@ -0,0 +1,126 @@
+// Copyright (c) 2019 by Robert Bosch GmbH. All rights reserved.
```

Comment:
Why is this a completely new file?

This file should be either `iceperf_follower` or `iceperf_leader`. I think you forgot to merge the latest master?! Or something like that...

Reply:
this file is long gone. It seems it's a bad merge

Reply:
Upsi, I'll check